pages/account/index.tsx

```diff
@@ -24,13 +24,13 @@ export default function MyAccount({
   async function checkoutRenew(checkoutId, patronId) {
     try {
       const response = await fetch(
-        `/research/research-catalog/api/account/checkouts/renewal/${checkoutId}`,
+        `/research/research-catalog/api/account/checkouts/renew/${checkoutId}`,
         {
           method: "POST",
           headers: {
             "Content-Type": "application/json",
           },
-          body: JSON.stringify(patronId),
+          body: JSON.stringify({ patronId: patronId }),
```

Comment:
Small nit but this and all the other same-name assignments can be shortened to:

JSON.stringify({ patronId })